src/screens/DangerPoints/styles.ts

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components/native';
+import { View } from 'react-native';
+
+export const CircleAttuation = styled.View`
+  border-width: 3;
+  border-color: #ff2600;
+  background-color: #ff3c007d;
+  height: 100px;
+  width: 100px;
+  border-radius: 100000;
```

Comment:
It lacked the measurement unit, when I tested it complained that the famous "px" was missing in all "border-radius" of this style.

`border-radius:100000px;`